src/java.desktop/share/native/libawt/java2d/loops/Blit.c

```diff
@@ -74,19 +74,16 @@ Java_sun_java2d_loops_Blit_Blit
 
     srcInfo.bounds.x1 = srcx;
     srcInfo.bounds.y1 = srcy;
-    if (UNSAFE_TO_ADD(srcx, width) ||
```

Comment:
isn't the MaskBlit_MaskBlit use the same pattern?

Reply:
But that doesn't cause any issue to the testcase given and existing testcase..Do you have any testcase which doesnt work because of MaskBlit pattern?

Reply:
I have not tried to create a test for MaskBlit, only for a simple Blit. I will take a look at the possibility of reproducing it.